server/graphql/v2/object/AccountStats.js

```diff
@@ -199,6 +201,180 @@ export const GraphQLAccountStats = new GraphQLObjectType({
           });
         },
       },
+      amountPledgedTimeSeries: {
+        description: 'Amount pledged time series',
+        type: new GraphQLNonNull(GraphQLTimeSeriesAmount),
+        args: TimeSeriesArgs,
+        /**
+         * @param {import('../../../models').Collective} account
+         */
+        async resolve(account, args) {
+          const dateFrom = args.dateFrom || moment().toDate();
+          const dateTo = args.dateTo || moment().add(24, 'month').toDate();
+          const timeUnit = args.timeUnit || getTimeUnit(getNumberOfDays(dateFrom, dateTo, account) || 1);
+
+          if (moment(dateFrom).isAfter(dateTo)) {
+            throw new ValidationFailed("'dateFrom' must be before 'dateTo'");
+          }
+
+          /**
+           * @type {{pledges: number; nextChargeAt: Date; currency: string; totalAmount: number}[]}
+           */
+          const currentMonthPledges = await sequelize.query(
+            `
+            SELECT
+                DATE_TRUNC('day', s."nextChargeDate") as "nextChargeAt",
+                count(1) as "pledges",
+                o."currency",
+                sum(o."totalAmount") as "totalAmount"
+            FROM "Orders" o
+            JOIN "Subscriptions" s on s.id = o."SubscriptionId"
+            WHERE TRUE
+            AND s."isActive"
+            AND s."nextChargeDate" > NOW()
+            AND s."nextChargeDate" <= DATE_TRUNC('month', NOW()) + interval '1 month' - interval '1 day'
```

Comment:
Maybe `currentMonthPledges` should be `currentTimeUnitPledges`, and the interval computed like:
- timeUnit: SECOND => `interval '1 second'`
- timeUnit: MINUTE => `interval '1 minute'`
- timeUnit: HOUR => `interval '1 hour'`
- timeUnit: DAY => `interval '1 day'`
- timeUnit: WEEK => `interval '1 week'`
- timeUnit: MONTH => `interval '1 month' - interval '1 day'`
- timeUnit: QUARTER => `interval '3 months' - interval '1 day'`
- timeUnit: YEAR => `interval '1 year' - interval '1 day'`

That way you'd be able to use this query to look at the same info with a focus on years rather than months.

```suggestion
            AND s."nextChargeDate" <= DATE_TRUNC('month', NOW()) + ${getIntervalFromTimeUnit()}
```